src/plot_api/plot_api.js

```diff
@@ -312,7 +312,7 @@ Plotly.plot = function(gd, data, layout, config) {
 
         // show annotations and shapes
         Registry.getComponentMethod('shapes', 'draw')(gd);
-        Registry.getComponentMethod('annoations', 'draw')(gd);
```

Comment:
Not sure. `Annotations.draw` gets called twice in `Plotly.plot`. Maybe one too many?